app/views/admin/divisions/_divisions_grid.html.slim

```diff
@@ -3,7 +3,7 @@
 
   - g.blank_slate do
     - @empty_list = true
```

Comment:
Empty list only happens when on page load there are no records returned from the initial filters (usually all records found).